win32ss/user/user32/controls/appswitch.c

```diff
@@ -481,11 +481,13 @@ BOOL ProcessHotKey(VOID)
 
       if (windowCount == 1)
       {
-         selectedWindow = 0;
-         CompleteSwitch(TRUE);
-         return TRUE;
+         MakeWindowActive(windowList[0]);
```

Comment:
Why using CompleteSwitch(TRUE) instead does not work?

Reply:
```c
if (!isOpen)
      return;
```
`CompleteSwitch` assume the open switcher.